src/simplewallet/simplewallet.cpp

```diff
@@ -2680,6 +2738,49 @@ bool simple_wallet::new_wallet(const boost::program_options::variables_map& vm,
   }
 
 
+  return true;
+}
+//----------------------------------------------------------------------------------------------------
+bool simple_wallet::new_wallet(const boost::program_options::variables_map& vm,
+    const std::string &multisig_keys, const std::string &old_language)
+{
+  auto rc = tools::wallet2::make_new(vm, password_prompter);
+  m_wallet = std::move(rc.first);
+  if (!m_wallet)
+  {
+    return false;
+  }
+
+  std::string mnemonic_language = old_language;
+
+  std::vector<std::string> language_list;
+  crypto::ElectrumWords::get_language_list(language_list);
+  if (mnemonic_language.empty() && std::find(language_list.begin(), language_list.end(), m_mnemonic_language) != language_list.end())
+  {
+    mnemonic_language = m_mnemonic_language;
+  }
+
+  m_wallet->set_seed_language(mnemonic_language);
```

Comment:
Does it make sense to deal with the mnemonic words when (to me at least) there doesn't seem to exist any intention to show multisig seed with words?

If the multisig seed is indeed intended to be shown with mnemonic words, I see quite a few inconsistencies, such as:

- simplewallet calls `m_wallet->get_multisig_seed(seed, seed_pass); ` with the third argument `raw` being defaulted to `true` which means that the mnemonic words will never be used.

- `set seed language` command refuses to proceed for multisig.

- gen_multisig should also let the user choose the seed language (currently it's left uninitialized).